packages/playground/src/App.vue

```diff
@@ -191,8 +215,26 @@ const routes: AppRoute[] = [
   },
   {
     title: "My Account",
-    items: [{ title: "Contracts", icon: "mdi-file-document-edit", route: "/contractslist" }],
+    items: [
+      {
+        title: "Contracts",
+        icon: "mdi-file-document-edit",
+        route: "/contractslist",
+      },
+    ],
   },
+  network !== "main"
+    ? {}
+    : ({
+        title: "Minting",
+        items: [
+          {
+            title: "Minting",
+            icon: "mdi-file-document-edit",
+            route: "/minting",
+          },
+        ],
+      } as any),
```

Comment:
```suggestion
  ...(network !== "main"
    ? []
    : [
        {
          title: "Minting",
          items: [
            {
              title: "Minting",
              icon: "mdi-file-document-edit",
              route: "/minting",
            },
          ],
        } as any,
      ]),
```

Adding empty object will throw an error